setenv.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# update the path
+export PATH=${HOME}/zopen/dev/depot_toolsport/depot_tools:$PATH
+
+# activate the virtual env
+. ./myenv/bin/activate
+
+# Disable updating depot_tools
+export DEPOT_TOOLS_UPDATE=0
```

Comment:
In my env I also have:
```
export DEPOT_TOOLS_BOOTSTRAP_PYTHON3=0
```
so `bootstrap_python3` doesn't run (by `ensure_bootstrap`; it's also run by `update_depot_tools` but the above envar already prevents it from running).

Reply:
Thanks for the update.  FWIW, I also removed the trailing spaces after one of the lines.  I could care less about such things, but I need to get back into the habit before we attempt to upstream.